example/index.html

```diff
@@ -96,6 +96,9 @@
           console.log('out of view', el, options);
         }
       }]);
+
+      window.addEventListener(ScrollTriggers.ScrollTriggersEvents.In, console.log);
```

Comment:
shouldn't this be `ScrollTriggers.Events?`

Reply:
That's what it exposes natively. Same below, the thing is we don't use scroll triggers like that and all the time the code is transpiled so it would be something like this:

`import ScrollTriggers from 'scroll-triggers'`
`import {default as ScrollTriggers, ScrollTriggersEvents} from 'scroll-triggers'`